package.json

```diff
@@ -3,6 +3,10 @@
   "version": "1.0.0",
   "description": "Backend for Bristol Tech Fair.",
   "main": "index.js",
+  "scripts": {
+    "build": "babel src --out-dir dist",
```

Comment:
Can we add script to restart application?

Reply:
I found a way to reload on changes after running `yarn start` by using `nodemon` and `babel-node`.

``` json
"start": "nodemon --exec babel-node -r dotenv/config ./src/server.js"
```

I think it's a cleaner approach and more user friendly than using `restart`,  Iet me know what you think 😃

Reply:
I found a way to reload on changes after running `yarn start` by using `nodemon` and `babel-node`.

``` json
"start": "nodemon --exec babel-node -r dotenv/config ./src/server.js"
```

I think it's a cleaner approach and more user friendly than using `restart`,  Iet me know what you think 😃

Reply:
That's great Shane. Its good idea to use nodemon in development env.